### DISCUSSION TITLE
Run Android with Windows 11

### DISCUSSION THREAD

**Discussion by Lorymi** [dotnet/maui#1457] [Q&A]
Hi😊 as we know windows 11 is coming. And it can run android app on windows (without external emulator)
So can we develop android app (maui) and run/debug on windows?

**Comment by Lorymi** [comment]
https://www.youtube.com/embed/VRvGe6lv7gA
Nice😋🤤

**Comment by dimonovdd** [comment]
https://www.youtube.com/watch?v=r3NwzDmmRdY&list=PLfbOp004UaYW2u0am9IhB_c3pWExLen42
